cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java

```diff
@@ -375,7 +376,7 @@ protected void run() throws Exception {
       TimeUnit.SECONDS.sleep(1);
     }
 
-    LOG.info("MapReduce Job is complete, status: {}, job: {}", job.isSuccessful(), context);
+    LOG.info("MapReduce Job completed{}. Job details: [{}]", job.isSuccessful() ? " with success" : "", context);
```

Comment:
Do you need a space after `completed`? Also, the message would look like `MapReduce Job completedtrue. Job details...`?